native_shortuuid/admin.py

```diff
@@ -1,34 +1,35 @@
+import django.conf
 import django.db.models
 
 import native_shortuuid
 
 
 class NativeUUIDSearchMixin:
-    def is_valid_shortuuid(self, seach_term):
+    search_uuid_fields = []
+
+    def is_valid_shortuuid(self, search_term):
         try:
             # It's not decode-able, thus, not a shortuuid
-            native_shortuuid.decode(seach_term)
+            native_shortuuid.decode(search_term)
         except ValueError:
             return False
 
         return True
 
     def get_search_fields(self, request):
         search_fields = super().get_search_fields(request)
-        if not hasattr(self, '_uuid_search_fields'):
-            self._uuid_search_fields = tuple(x for x in search_fields if x.endswith('uuid'))
-            self._non_uuid_search_fields = tuple(x for x in search_fields if not x.endswith('uuid'))
-        return self._non_uuid_search_fields or self._non_uuid_search_fields + ('id', )
+        if getattr(django.conf.settings, 'ADMIN_EXTRACT_UUID_SEARCH_FIELDS', False):
```

Comment:
not getattr(django.conf.settings, 'DISABLE_EXTRACT_SEARCH_UUID_FIELDS', False)
We want it to work by default, we don't need to add and extra settings for it to work. so just reverse the logic

Reply:
This is for backward compatibility, so the default shouldn't be the extract, I did that so you can fix it on your side

Reply:
@laithfawwaz I don't think it's a good idea when I wanna update the library then I need to add this settings to make it, it should work by default and if I wanna disable it I can do that using the settings. cc: @waseem-omar WDYT?

Reply:
@hadayda Once anyone updates the library, he should set in the base settings this variable and done, no need for any other to do the same thing, so this variable just tells the library `Hey, I use the old version, so don't break my code`

Reply:
@laithfawwaz BTW, don't forget to update the readme file about the new setting

Reply:
@laithfawwaz @hadayda if we removed this setting check here, what is going to break when we install the new version?

Reply:
@laithfawwaz I don't think removing the settings  will break the system right? if so I don't we need the backward compatibility at all

Reply:
@hadayda @waseem-omar this is for `backward compatibility` if LG team updated the version without this, they'll need to define this field `search_uuid_fields` when you want to search on uuids and remove the uuid fields from `search_fields` array

Reply:
@hadayda We wasn't documenting the `NativeUUIDSearchMixin` so no need to document this setting, however, I updated the readme to include how to use the admin mixin